src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,124 @@
+package frc.robot.subsystems.climber;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Climber extends SubsystemBase {
+    private final TalonFX master = new TalonFX(Ports.Climber.MASTER);
+    private final TalonFX slave = new TalonFX(Ports.Climber.SLAVE);
+    private final UnitModel unitModel = new UnitModel(Constants.Climber.TICKS_PER_METER);
+    private final Solenoid stopper = new Solenoid(Ports.Climber.STOPPER);

```

Comment:
Today i talked to liron and we are switching to double solenoids instead of solenoids